changelog.d/8164.misc

```diff
@@ -0,0 +1 @@
+Add functions to `MultiWriterIdGen` used by events stream.
```

Comment:
Not sure we need to do it now, but I wonder if these changelogs should be updated at some point to refer to a more user-focused result.